codeql_javascript_dataset/CWE-502/examples/README.md

```
@@ -0,0 +1,11 @@
+# CWE-502: Deserialization of Untrusted Data
+
+## Description
+Deserialization of Untrusted Data
+
+## Examples
+This directory contains examples demonstrating Deserialization of Untrusted Data vulnerabilities and their fixes.
+
+### Files:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)